zeeguu/core/model/user.py

```diff
@@ -727,11 +727,6 @@ def authorize_anonymous(cls, uuid, password):
     def create_default_user_preference(self):
         from zeeguu.core.model.user_preference import UserPreference
 
-        existing_preference = UserPreference.find_or_create(
-            db.session, self, UserPreference.PRODUCTIVE_EXERCISES
+        UserPreference.find_or_create(
+            db.session, self, UserPreference.PRODUCTIVE_EXERCISES, "true"
```

Comment:
we simplified this a bit - if you pass the value, you already have the code initializing the preference accordingly